webapp/src/main/resources/application.yml

```diff
@@ -89,6 +89,10 @@ permissionservice:
   endpoint: ${rdw.permission.service}
 
 app:
+  reporting:
+    webapp:
+      url: /
```

Comment:
Good point.
I realized actually now that you wrote this I need to change the variable name to be ```home-url``` or something. The idea here is that clicking the logo will by default take you to the home page of the admin app OR if configured to, take you back to the reporting webapp. And in the default case it's best to use "/" so we can take advantage of the browser's host fill-in